From b3c80dc4a569a9c06319fd1cb3d1ce992f45f20f Mon Sep 17 00:00:00 2001
From: "Dr. Greg" <greg@enjellic.com>
Date: Wed, 10 Feb 2021 18:58:51 -0600
Subject: [PATCH 17/50] Rename events filesystem initialization function.

This commit converts the function that initializes the
/sys/kernel/security/integrity/events pseudo-filesystem from the
following:

ima_fs_init_identity

To the following:

ima_fs_init_identity
---
 security/integrity/ima/ima.h        | 2 +-
 security/integrity/ima/ima_events.c | 2 +-
 security/integrity/ima/ima_fs.c     | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/security/integrity/ima/ima.h b/security/integrity/ima/ima.h
index abe929807f04..0abee48f59cf 100644
--- a/security/integrity/ima/ima.h
+++ b/security/integrity/ima/ima.h
@@ -175,7 +175,7 @@ int ima_events_is_mapped(int, struct integrity_iint_cache *iint,
 			 struct file *file, const char *pathname,
 			   struct ima_template_desc *, int);
 int ima_fs_init(void);
-struct dentry *ima_fs_init_identity(void);
+struct dentry *ima_fs_init_events(void);
 
 /*
  * used to protect h_table and sha_table
diff --git a/security/integrity/ima/ima_events.c b/security/integrity/ima/ima_events.c
index 458abbd661f1..25576121020a 100644
--- a/security/integrity/ima/ima_events.c
+++ b/security/integrity/ima/ima_events.c
@@ -1663,7 +1663,7 @@ int __init ima_events_init(void)
 {
 	int retn = -1;
 
-	events_dir = ima_fs_init_identity();
+	events_dir = ima_fs_init_events();
 	if (events_dir == NULL)
 		goto done;
 
diff --git a/security/integrity/ima/ima_fs.c b/security/integrity/ima/ima_fs.c
index fdd1a4d18b22..4ddbfb432f4a 100644
--- a/security/integrity/ima/ima_fs.c
+++ b/security/integrity/ima/ima_fs.c
@@ -449,7 +449,7 @@ static const struct file_operations ima_measure_policy_ops = {
 	.llseek = generic_file_llseek,
 };
 
-struct dentry * __init ima_fs_init_identity(void)
+struct dentry * __init ima_fs_init_events(void)
 {
 	struct dentry *id_dir = NULL;
 
-- 
2.31.1

